### DISCUSSION TITLE
Parent Beta Reconstruction in Titanium

### DISCUSSION THREAD

**Discussion by ly801** [mtex-toolbox/mtex#1713] [Ask Anything]
Dear all,

May I ask whether the 'parent beta reconstruction' codes also work for the martensitic alpha prime phase in titanium alloys? Both the primary alpha and alpha prime phases have hcp structure, but with slightly different lattice parameters. Are both their symmetry types '622'? Thank you!

Best regards,
Lu.

**Comment by AdamLeaghj** [comment]
Hi, I think they have a separate forum for ORTools just in case you don't know. https://github.com/ORTools4MTEX/ORTools/discussions. You can post your question there.

**Comment by Savillian** [comment]
Hi Lu!

I've had success reconstructing both alpha and alpha prime microstructures in titanium. Be careful with the 622 space group symmetry though - both those crystal structures are actually 6/mmm. I do not know why many programs assume alpha titanium is 622, but if you make the IPF triangle color keys you'll see a noticeable difference.

Same thing with beta titanium. Places list it as 432, but it's really 4/mmm... The number of incorrect IPF maps I've seen because of this is alarmingly high.
